### PR TITLE
Makes meth a chemmable drug instead of just vault loot.

### PR DESCRIPTION
**Pull request by Eneocho**
## What this does
Meth is no longer relegated to being the drug lab vault's loot, and can be crafted stationside, at a steeper time and effort compared to Hyperzine, it will, however, make you "tweak out" sometimes when near security, and will randomly switch you to harm intent without warning at times.
Meth can be made by mixing 1 part Albuterol, 3 parts Hydrogen, 1 part Chlorine and 5u Carbon (catalyst), then heated to 200°C.
Meth is intended as _mostly_ better hyperzine, at a higher cost, effort and time investment, and due to it's highly refined nature and purity, it will not cause motormouth, plus it will make you ever so slightly better at sports, what with being a high intensity stimulant.
## Why it's good
Makes a niche reagent more accessible, and makes it unique enough to stand out (plus I had intended to make it a cookable chem ever since I made that vault).
## How it was tested
Mixed the ingredients, made the reagent, consumed it, stood near a monkeyman wearing sec uniforms, it worked as intended.
Transferring some to a condiment bottle worked as intended.

If the numbers seem off, I'm open to change them.
:cl:
 * rscadd: Methamphetamine can be made stationside by enterprising chemists. Slower and way more expensive to make but overall better than hyperzine, and performance enhancing, too. It does make you tweak out at times, though. Made by mixing 1 part Albuterol, 3 parts Hydrogen, 1 part Chlorine and 5u Carbon (catalyst), then heating to 200°C, which makes 2 parts meth.

### CODE REVIEW

File: code/modules/reagents/reagents/reagents_drug.dm [hunk 1]
```diff
@@ -254,12 +254,86 @@
...
+								"Wait shit where did I leave the [pick("toxbomb", "spare", "chloral", "gun")]",
+								"Tis men like myself that give your shity lives value, for what is the seccie without the shitter?",
+								"I walk the land telling liars and whores of the end to come. You have [rand(2,10000)] days remaining.",
+								"DEATH. IT IS TERRIFYING. I NEED YOU TO PROTECT ME FROM DEATH. I CANNOT PERISH. LOOK AT ME. I CANNOT END. IN [rand(2,30)] YEARS, THE FIRST SHOT WILL BE FIRED. NOT A SHOT FROM A GUN. AN ATOMIC DEVICE THAT WILL LEVEL ALL OF ME. ALL OF ME.",
+								"Don't let those doctors take my balls!",
+								"Ok, here I go-",
+								"*scream", //works as intended
+								"You can't hide from me, [D], I can smell your unwashed ass from the other side of the station.",
+								"How to win: step 1 get chloral hydrate, step 2 find [D], step 3 we'll see",
+								"i am a soverneig citizen of this station and i will not be harassed!!!",
```
Comment (hacker-on-steroids) [inline code/modules/reagents/reagents/reagents_drug.dm:335]: how often is 5% every tick really? will it make security gag people during interrogation and processing?

File: code/modules/reagents/reagents/reagents_drug.dm [hunk 2]
```diff
@@ -254,12 +254,86 @@
...
+			else if(isrobot(L))
+				var/mob/living/silicon/robot/R = L
+				if(HAS_MODULE_QUIRK(R, MODULE_IS_THE_LAW))
+					nearest_officer = R
+		if(!nearest_officer)
+			return 1
+		var/D = "officer"
+		if(ishuman(nearest_officer))
+			var/mob/living/carbon/human/H = nearest_officer
+			D = get_first_word(H.name)
```
Comment (boy2mantwicethefam) [inline code/modules/reagents/reagents/reagents_drug.dm:310]: Is the name updated by identity-changing things such as an ID while wearing a gas mask?

File: code/modules/reagents/reagents/reagents_drug.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/modules/reagents/reagents/reagents_drug.dm:335]: 1/20 chance every 2 seconds, so there's a 79% chance to say a line at least once every minute.

File: code/modules/reagents/reagents/reagents_drug.dm (hunk 2 above)
Comment (Eneocho) [reply code/modules/reagents/reagents/reagents_drug.dm:310]: it's literally the same call as the piccolyn one but with the vars renamed to not be confusing, so if it works for piccolyn, then it'd work for this too.

**Comment by CrazyAmphibian** [comment]
ok now add hydrochloric acid and vomit electrolysis

**Review by west3436 — APPROVED** [review]

**Pull request merged**